bodo/build.gradle.kts

```diff
@@ -0,0 +1,72 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to you under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import com.github.autostyle.gradle.AutostyleTask
+
+plugins {
+    id("com.github.vlsi.ide")
+    calcite.fmpp
+    calcite.javacc
+}
+
+dependencies {
+    api(project(":core"))
+    api(project(":babel"))
```

Comment:
If babel is required do we still need to install it directly in the `pom.xml` for BodoSQL? Is it not implicitly captured in the generated JAR file?

Sorry this is not directly related to the changes in this PR but also the Bodo PR.